### ISSUE TITLE
[BUG / PROBLEM] [NUTSCRIPT 1.0] Saving issue or something with Doors, Areas and Spawns.

### ISSUE THREAD

**Issue by Proxykins** [Chessnut/NutScript#305]
Nutscript 1.0

Hi, I cant provide much evidence as its a common sense thing to note:
My doors, areas and spawn points for factions and classes don't save / or at least there not there when I restart, and I have tried different restarts and everything. I use the SQLite as I am not advanced enough for the MySQL business. Do you think you can shine light on the subject I really want to get started on Class making not resetting doors / areas or spawn points all the time.

Thanks
Proxykins

**Comment by Proxykins** [comment]
Looking a others I notice, you just close it if you cant replicate it, so I am going to prevent this by saying. It cant be replicated due to it being not an error, not a piece of code, but just something that happened ever since I started it up from fresh download. Soooo yeah, please respond ASAP.

**Comment by Chessnut** [comment]
Make sure you shut down the server properly using the `quit` command. Check if the data/nutscript folder is writable.

**Comment by DeadiKation** [comment]
Actually this happens even after shutting down the server correctly. Sometimes it happens after several restarts, other times it happens after 1 over and over. It's random and seems to wipe the data.

**Comment by Chessnut** [comment]
It may be that you are restarting before the data is loaded.

**Comment by DeadiKation** [comment]
Well I tested this specifically, and it turns out to be sort've random. I tried 5 times, all with the same restarts, allowing idle for about 15 minutes each. About 3 times it kept the same things saved, then it would wipe them after the 4th restart, I didn't set anything new and let it idle for 30 minutes, hoping the data would load, Before i restarted I checked where the data was saving, hoping to see it there. It was empty showing [[]] as the information inside.

I've done a few more tests since then, At times I can see perfectly the data is there. Other times I see [[]] meaning all the data inside of it was wiped. Might be a loading issue when attempting to load it flushes the table?
